Doppler.ContactPolicies.Api.Test/UpdateContactPoliciesSettingsTest.cs

```diff
@@ -34,29 +32,27 @@ public UpdateContactPoliciesSettingsTest(WebApplicationFactory<Startup> factory)
         }
 
         [Theory]
-        [InlineData("test1@test.com", CONTACT_POLICIES_SETTINGS, TOKEN_ACCOUNT_123_TEST1_AT_TEST_DOT_COM_EXPIRE_20330518,
+        [InlineData(CONTACT_POLICIES_SETTINGS_REQUEST_BODY_STUB, "test1@test.com", TOKEN_ACCOUNT_123_TEST1_AT_TEST_DOT_COM_EXPIRE_20330518,
             HttpStatusCode.NotFound)]
         public async Task
             UpdateContactPoliciesSettings_Should_ReturnNotFound_When_UserWithSameAccountNameIsNotFound(
-                string accountName, string contactPoliciesRequestBody, string token, HttpStatusCode expectedStatusCode)
+                string contactPoliciesRequestBodyStub, string validAccountName, string token, HttpStatusCode expectedStatusCode)
         {
             // Arrange
             int? expectedNotFoundedIdUser = null;
 
             var contactPoliciesMock = new Mock<IContactPoliciesService>();
-            contactPoliciesMock.Setup(x => x.GetIdUserByAccountName(accountName)).ReturnsAsync(expectedNotFoundedIdUser);
+            contactPoliciesMock.Setup(x => x.GetIdUserByAccountName(It.IsAny<string>())).ReturnsAsync(expectedNotFoundedIdUser);
```

Comment:
Se puede dar que dentro del método de la controladora `UpdateContactPoliciesSettings` llamemos a `GetIdUserByAccountName` con un `accountName` diferente al del endpoint (`/accounts/{accountName}/settings`)? Entiendo que deberíamos utilizar lo que nombraste como `validAccountName`, no? Cualquier cosa lo hablamos 
```suggestion
            contactPoliciesMock.Setup(x => x.GetIdUserByAccountName(validAccountName)).ReturnsAsync(expectedNotFoundedIdUser);
```